enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,220 @@
+
+### Examples
+
+Example for a placement with placementStrategy has canary cluster defined and 50% max concurrency.
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: operator-placement
+  namespace: ztp-acm-ns
+spec:
+  predicates:
+    - requiredClusterSelector:
+        labelSelector:
+          matchExpressions:
+            - key: common-profile
+              operator: In
+              values:
+                - 'true'
+  placementStrategy:
+    canaryClusters:
+      matchExpressions:
+        - key: canary
+          operator: Exist
+    clustersGroupPercentage: 50%
+status:
+  numberOfSelectedClusters: 300
+  numberOfGroups: 3
+  conditions:
+    - lastTransitionTime: '2023-02-16T18:32:05Z'
+      message: All cluster decisions scheduled
+      reason: AllDecisionsScheduled
+      status: 'True'
+      type: PlacementSatisfied
+```
+
+With 300 clusters selected based on the placements predicates and the defined placement strategy there will be 3 cluster groups presented in the placementDecisions; 1) Canary cluster group has ClusterGroupIndex=0 2) Cluster group 1 has ClusterGroupIndex=1 and 3) Cluster group 2 has ClusterGroupIndex=2. The created placementDecision should be as below.
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: PlacementDecision
+metadata:
+  name: operator-placement-decision-canary
+  namespace: ztp-acm-ns
+  labels:
+    cluster.open-cluster-management.io/placement: operator-placement
+    cluster.open-cluster-management.io/clustergroupindex: 0
+status:
+  decisions:
+    - clusterName: cls1
+      reason: ''
+    ...
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: PlacementDecision
+metadata:
+  name: operator-placement-decision-1
+  namespace: ztp-acm-ns
+  labels:
+    cluster.open-cluster-management.io/placement: operator-placement
+    cluster.open-cluster-management.io/clustergroupindex: 1
+status:
+  decisions:
+    - clusterName: cls5
+      reason: ''
+    ...
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: PlacementDecision
+metadata:
+  name: operator-placement-decision-3
+  namespace: ztp-acm-ns
+  labels:
+    cluster.open-cluster-management.io/placement: operator-placement
+    cluster.open-cluster-management.io/clustergroupindex: 2
+status:
+  decisions:
+    - clusterName: cls155
+      reason: ''
+    ...
+```
+
+The Placement Helper [library](https://github.com/open-cluster-management-io/api/blob/main/cluster/v1beta1/helpers.go) should provide functions to facilitate retrieve the cluster groups.
+
+
+### How Workload applier APIs will benefits from using placementStrategy
```

Comment:
I think we should have some examples.

Reply:
add example for manifestWorkReplicaSet use of the proposed placement